public/locales/de/translation.json

```
@@ -1,4 +1,7 @@
 {
+  "InfoBar": {
+    "ReturnToFocusedNote": "Zurück zur Fokuskarte"
```

Too much hokus ~~pokus~~ fokus :P 
```suggestion
    "ReturnToFocusedNote": "Zurück zur fokussierten Karte"
```
Or do you think it's too long? I think the word "Fokuskarte" is not selfexplanatory for the most users. If you agree and have a better translation, go for it. If not, then it's fine too.

---

It's too long, that's why I've used 'Fokuskarte'